packages/manager/apps/container/src/container/nav-reshuffle/sidebar/SidebarLink.tsx

```diff
@@ -80,47 +83,57 @@ type SidebarLinkProps = {
   count?: number | boolean;
   node?: Node;
   linkParams?: Record<string, string>;
-  onClick?(): void;
+  handleNavigation?(): void;
   id?: string;
+  short?: boolean;
 };
 
 const SidebarLink: React.FC<ComponentProps<SidebarLinkProps>> = ({
   count = 0,
   node = {},
   linkParams = {},
-  onClick = () => { },
+  handleNavigation = () => {},
   id = '',
+  short = false,
 }: SidebarLinkProps): JSX.Element => {
   const { t } = useTranslation('sidebar');
+  const mobile = isMobile();
 
   return !node.children && (node.url || node.routing) ? (
     <StaticLink
-      onClick={onClick}
+      handleClick={handleNavigation}
       count={count}
       node={node}
       linkParams={linkParams}
       id={id}
+      short={short}
     />
   ) : (
-    <a onClick={onClick} id={id}>
-      {t(node.translation)}
-      {node.children ? (
+    <div
+      onMouseOver={!mobile ? handleNavigation : null}
```

Comment:
When in "short mode" the hover is only trigger on the text (and not the whole div) can you take a look please